### DISCUSSION TITLE
not working on unlockd.me

### DISCUSSION THREAD

**Discussion by mostlyhuman** [aclap-dev/video-downloadhelper#496] [Help and Q&A]
### Your question

not working on unlockd.me

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "microsoft",
  "channel": "stable",
  "page_url": "https://unlockd.me/thrashur?tab=feed&subTab=video",
  "error": {
    "details": "SideDownload error: : [tcp @ 0000020f030de280] Connection to tcp://vz-9f7b547e-aeb.b-cdn.net:443 failed: Error number -138 occurred\r\n[in#0 @ 0000020f0499b700] Error opening input: Error number -138 occurred\r\nError opening input file https://vz-9f7b547e-aeb.b-cdn.net/aef90a77-37e0-4c94-8951-c024e0a804d8/720p/video.m3u8?token=npO9hRMsy9ZXjNIo2S5HhUSHcCSq27puzYUHOd8KW_E&expires=1715788093.\r\nError opening input files: Error number -138 occurred\r\n",
    "downloadable_id": "downloadable_7137787991601632",
    "error": "coapp_failure",
    "id": "coapp_failure_22b5e28e-54fb-4a7d-b88c-01d67074003f"
  }
}

**Comment by paulrouget** [comment]
- Did this used to work before or it's a recent regression?
- is there a free video example on that website then I can test locally?

**Comment by mostlyhuman** [comment]
Not sure if worked before or not, there are a lot of videos you can view for free but might have to register a free account, not sure on that.

**Comment by lmwvieira-beep** [comment]
any update on this?

**Reply by Mattmjs** [reply]
It should work now on the latest version.

**Reply by lmwvieira-beep** [reply]
No, I have the latest version, but still no luck.

**Reply by Mattmjs** [reply]
Version 10 ?

**Reply by lmwvieira-beep** [reply]
v10.0.271.2

**Reply by GrampaWildWilly** [reply]
I visited the web page whose URL appears in your original post.  This looks like a very expensive porn site.  I would imagine that you are utterly out of luck on this one.  Nobody is going to sign up for $25 per month just to see if VDH works there.  You've apparently done the basic grunt work & shown that the new VDH 10 does not work there.  All I can offer is that you try my `Plan B`.  You can discover what that is by searching this forum using that as you search key.  Put quotation marks around it.
